test/test_helper.exs

```diff
@@ -16,13 +16,27 @@ Application.put_env(:phoenix_live_reload, MyApp.EndpointScript, [
   url: [path: "/foo/bar"],
 ])
 
+Application.put_env(:phoenix_live_reload, MyApp.EndpointScoped,
+  live_reload:
+    [url: "ws://localhost:4000",
+     scope: "/foo/bar",
```

Comment:
So a two "paths" scope is not going to work because you are matching only a single prefix here:

https://github.com/phoenixframework/phoenix_live_reload/pull/96/files#diff-c8be1eead197c8e20eb612e511f68033R72

Perhaps it would be best to add the scope at the end of the path? i.e. "/phoenix/live_reload/foo/bar" instead of "/foo/bar/phoenix/live_reload"?

Reply:
Hmmm yeah I see what you mean but then it's not a scope 😄 

And it will also mean that you need to define it differently in `apps/app_one/lib/app_one_web/endpoint.ex`:
```elixir
if code_reloading? do
  socket "/phoenix/live_reload/socket/app_one", Phoenix.LiveReloader.Socket
  ...
end
```

Maybe it's better then to support only one path scope?

Reply:
We can call it "suffix" then?

Reply:
Hmmm yeah that could work, I just find it that the word scope makes it more clear of what it actually does. I find suffix to be a bit more ambiguous as to what exactly the effect of this configuration will do...

Reply:
I see your point but it is not really a scope either. It is just the path suffix we want to access the configuration at.

Also, if you have two endpoints... wouldn't they be running at different ports and so on?

Reply:
Maybe I should explain my situation a bit better...

Yes they are running at different ports and I'm using https://github.com/jesseshieh/master_proxy for proxying using the `path` configuration.

Because the live reload path is same for both apps (and not scoped according to the proxy rules I made) `/phoenix/live_reload/socket` always get proxied to the default app.

That's why I went first with the `scope` idea but I could change the proxy rule to work also with suffix. I'm testing it now and if it works I'll commit it and will merge all the commits into one and force push.

Reply:
Hmmm, it doesn't work... I'm getting a 404 on `ws://localhost:5000/phoenix/live_reload/socket/app_one/websocket?vsn=2.0.0`

I'm running the proxy on port 5000.

Any ideas?

Reply:
These are the changes I made:

`apps/app_one/config/dev.exs`:
```elixir
config :app_one, AppOneWeb.Endpoint,
  live_reload: [
    suffix: "/app_one",
    patterns: [
      ~r"priv/static/.*(js|css|png|jpeg|jpg|gif|svg)$",
      ~r"priv/gettext/.*(po)$",
      ~r"lib/app_one_web/{live,views}/.*(ex)$",
      ~r"lib/app_one_web/templates/.*(eex)$"
    ]
  ]
```

`apps/app_one/lib/app_one_web/endpoint.ex`:
```elixir
if code_reloading? do
  socket "/phoenix/live_reload/socket/app_one", Phoenix.LiveReloader.Socket
  ...
end
```

Reply:
Ok ignore my comments about it not working, I guess somehow I got stuff mixed up 😆 

I'm doing some extra testing and I'll push the suffix change 😉